netbox/extras/models/mixins.py

```diff
@@ -125,12 +126,22 @@ def get_context(self, context=None, queryset=None):
             class_name=self.__class__
         ))
 
+    def get_environment_params(self):
+        """
+        Pre-processing of any defined Jinja environment parameters (e.g. to support path resolution).
+        """
+        params = self.environment_params or {}
+        for name, value in params.items():
+            if name in JINJA_ENV_PARAMS_WITH_PATH_IMPORT and type(value) is str:
+                params[name] = import_string(value)
+        return params
```

Comment:
Couple of questions here:

1. Should we do any validation of `value`? E.g. make sure it's within a set of accepted values?
2. I assume `import_string()` might raise some exceptions, especially around dotted-paths that don't exist. Is the intention to let those bubble up to the user?

Reply:
Good point.